constants.js

```diff
@@ -1,6 +1,6 @@
 
 const PG_HOST = process.env.PG_HOST || '127.0.0.1';
-const PG_PORT = process.env.PG_PORT || 45775;
+const PG_PORT = process.env.PG_PORT || 5432;
```

Comment:
I believe the default port is 45775, please revert and do not change the default port